stable/jenkins/templates/jenkins-master-servicemonitor.yaml

```diff
@@ -18,7 +18,11 @@ spec:
   endpoints:
   - interval: {{ .Values.master.prometheus.scrapeInterval }}
     port: http
+    {{- if .Values.master.ingress.path }}
+    path: {{ .Values.master.ingress.path }}{{ .Values.master.prometheus.scrapeEndpoint }}
```

Comment:
My first impression was that there is a "/" missing between both values, but it's correct as the scrapeEndpoint is by default `/prometheus`

Checking the `values.yaml` for the ingress:
```
  ingress:
    # Set this path to jenkinsUriPrefix above or use annotations to rewrite path
    # path: "/jenkins"
```
I feels it's better to use `.Values.master.jenkinsUriPrefix` instead of `.Values.master.ingress.path`. As exposing Jenkins via Ingress is no pre-condition to be able to scrape metrics from running pods.